### ISSUE TITLE
[Feature request] Optimize "screenshot" and "snapshot"

### ISSUE THREAD

**Issue by basuritashka** [ValveSoftware/halflife#2271]
I just took a screenshot (using `screenshot` and `snapshot`) and it threw a .tga/.bmp file with a 6MB size (and the `screenshot` command threw a .tga file, rare extension nowadays)

`screenshot` output
![photo](https://user-images.githubusercontent.com/33007491/56699168-6ecb0c00-66c2-11e9-89da-5fa105a579b2.png)

![imagen](https://user-images.githubusercontent.com/33007491/56699250-c7020e00-66c2-11e9-905e-a31574d6c923.png)

We should be getting a lower size image and not a 6mb one.

We can also implement a new cvar like on CS:GO `cl_steamscreenshots` and just let Steam take the photo.

**Comment by Sockman1** [comment]
Seems like a redundant change when you can just take a screenshot through steam with compression. The space of one or a few uncompressed images isn't a huge concern for modern computers anyway.

**Comment by basuritashka** [comment]
> 
> 
> Seems like a redundant change when you can just take a screenshot through steam with compression. The space of one or a few uncompressed images isn't a huge concern for modern computers anyway.

I know, but these commands exists on the game, or we delete them or we just optimize them as they are throwing a 6mb file on each usage.

**Comment by Sockman1** [comment]
Steam has an option to save uncompressed versions of screenshots. Because of the nature of bmp and tga files, they're going to be about 6mb for a 1920x1080 image. If you want something of high quality, these are the best you're going to get.

**Comment by basuritashka** [comment]
True, but then `snapshot `and `screenshot `command should be deleted from the game as they are useless.

**Comment by Sockman1** [comment]
They can be useful for mods. tga and bmp files are pretty much the only image files used by goldsrc, and the commands are more convenient that getting an uncompressed image from steam and converting it. For example, you can take screenshots of a map and use that as a skybox texture.
There's not much point in removing something anyway, unless there's some exploit attached.
